algo/src/main/java/com/hartwig/serve/dao/ServeDAO.java

```diff
@@ -218,7 +219,7 @@ private static void writeActionableHotspotBatch(@NotNull Timestamp timestamp, @N
                 actionableHotspot.applicableCancerType().name(),
                 actionableHotspot.applicableCancerType().doid(),
                 concat(toStrings(actionableHotspot.blacklistCancerTypes())),
-                actionableHotspot.description(),
+                actionableHotspot.efficacyDescription(),
```

Comment:
You don't seem to write the year?

This seems to be cause in a number of functions in this class?